projects/core/src/anonymous-consents/store/effects/anonymous-consents.effect.ts

```diff
@@ -27,6 +28,8 @@ export class AnonymousConsentsEffects {
     AnonymousConsentsActions.LoadAnonymousConsentTemplates
   > = this.actions$.pipe(
     ofType(SiteContextActions.LANGUAGE_CHANGE, AuthActions.LOGOUT),
+    // TODO(issue:4989) Anonymous consents - remove this filter
+    filter(_ => isFeatureLevel(this.anonymousConsentsConfig, '1.3')),
```

Comment:
same genuine question, why use the utils and not the service (same thing thing since in service just calls that)